<launch>
  <arg name="real_machine" default="false" />
  <arg name="auto_takeoff" default="false" />
  <arg name="joint_control_topic_name" />

  <node pkg="bspline_generator" type="aerial_plannar_node" name="aerial_plannar" output="screen">
    <param name="debug_output" value="false" />
    <param name="control_frequency" value="100.0" />

    <!-- trajectory relate -->
    <param name="bspline_degree" value="5" />
    <param name="trajectory_period" value="50.0" />

    <param name="target_offset_x" value="1.4" />
    <param name="target_offset_y" value="0.0" />

    <param name="manual_start_state" value="true" />
    <param name="start_x" value="-1.4" />
    <param name="start_y" value="0.17" />
    <param name="start_yaw" value="-1.57" />

    <!-- trajectory tracking relate -->
    <param name="joint_upperbound" value="1.58" />
    <param name="joint_lowerbound" value="-1.58" />
    <param name="joint_control_topic_name" value="$(arg joint_control_topic_name)" />

  </node>
</launch>
